test/deveryRegistryTokenization.js

```diff
@@ -174,7 +173,26 @@ contract('DeveryRegistry - ERC721 - tokenization tests', (accounts) => {
       console.log('the transasction was a success (which means things went wrong)');
     }).catch(err => {
       hasTransactionFailed = true
-    })
+    });
     assert.equal(hasTransactionFailed, false, "The transaction didn't failed, something went wrong when setting the maximum mintable quantity");
+  });
+
+  it('should check if an account has a specific product', async () => {
+    //Will use two accounts, one of them will have the specific product
+    //The other won't
+
+    const deveryERC721Instance = createDeveryERC721(web3, undefined, myAccount, deveryERC721Contract.address);
+    
+    const withProductAccount = myAccount;
+    const withoutProductsAccount = accounts[1];
+    //TODO: remove magic strings and use values created inside this test
+    //instead of leveraging from values that come from side effects of other tests
+    const productAddress = '0xde266616A37934Abe076816CBc89256B499004E2';
```

Comment:
Where this value comes from? if if changes in its source place this is going to break this test.